src/history/git.js

```diff
@@ -1,30 +1,27 @@
 import fs from 'fs';
 import path from 'path';
-import git from 'isomorphic-git';
+
+import dotenv from 'dotenv';
+dotenv.config();
+import simpleGit from 'simple-git';
 
 const __dirname = path.dirname(new URL(import.meta.url).pathname);
 
-const DEFAULT_GIT_OPTIONS = {
-  fs,
-  dir: path.resolve(__dirname, '../..'),
-  dryRun: process.env.NODE_ENV === 'test',
-};
+export const git = simpleGit(path.resolve(__dirname, '../..'));
+
+if (process.env.CI) {
```

Comment:
Because it changes the global git config, thus all your next commits will be attributed to AmbaNum Bot. But I can't remove it neither as on the CI git is not configured and all tests failed because it can't commit.